modules/backend/controllers/Auth.php

```diff
@@ -90,8 +90,8 @@ public function signin_onSubmit()
             throw new ValidationException($validation);
         }
 
-        if (($remember = config('cms.backendForceRemember', true)) === null) {
-            $remember = (bool) post('remember');
+        if (($remember = Config::get('cms.backendForceRemember', true)) === null) {
+            $remember = (bool)post('remember');
```

Comment:
revert the space after type cast change please

Reply:
And @Samuell1 can you make the tests pass by fixing the other coding style issues in this file? Right now it's just the one liner conditional in filterList() that needs to be addressed AFAIK